.github/integration/scripts/charts/dependencies.sh

```diff
@@ -31,7 +31,7 @@ else
 fi
 
 # secret for the crypt4gh keypair
-C4GHPASSPHRASE="$(random-string)"
+C4GHPASSPHRASE="ZB8ko3g2P9wEgagpgqCS7vJhJoKrSUfJ"
```

Comment:
```suggestion
C4GHPASSPHRASE="$(random-string)"
```

There is no reason for this change

Reply:
I also prefer no changes.

Reply:
@nanjiangshu  If we don't use a fixed value here then  due to this block [here](https://github.com/neicnordic/sensitive-data-archive/blob/f1979be39c23318500ba58cb340608689431557c/.github/integration/scripts/charts/dependencies.sh#L40), you need to manually remove the keys every time you want to run local tests for Helm charts using the commands from the Makefile. 

But I'll set it back to random and make an issue.

Reply:
Change reverted, new issue opened regarding this: #1208.

Reply:
Aha, I don't know about this. That's a good point.

Reply:
That's on me, I should have explained it better :smile: .